packages/generator/src/service/package-json.ts

```diff
@@ -46,15 +46,15 @@ export async function packageJson(
             : {})
         },
         dependencies: {
-          '@sap-cloud-sdk/odata-common/internal': `^${await getSdkVersion()}`,
+          '@sap-cloud-sdk/odata-common': `^${await getSdkVersion()}`,
```

Comment:
Here is a potential candidate of missing test coverage.

Reply:
Done!